### DISCUSSION TITLE
How to align accordion in form.

### DISCUSSION THREAD

**Discussion by pkk-code** [fyne-io/fyne#4055] [Q&A] [answered]
How to left align the accordion to the form so that the detail items are listed. Right now, it's off to the 2nd column.  Please let me know if there is a better way to include the accordion in the form and properly aligned.

```go
package main

import (
	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/container"
	"fyne.io/fyne/v2/layout"
	"fyne.io/fyne/v2/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Accordion - with form")
	w.Resize(fyne.NewSize(500, 500))
	entry := widget.NewEntry()
	textArea := widget.NewMultiLineEntry()
	form := &widget.Form{
		Items: []*widget.FormItem{
			{Text: "Name", Widget: entry}, {Text: "Description", Widget: textArea}},
		OnSubmit: func() {

			w.Close()
		},
	}
	label1 := widget.NewLabel("City")
	value1 := widget.NewEntry()
	label2 := widget.NewLabel("State")
	value2 := widget.NewEntry()
	grid := container.New(layout.NewFormLayout(), label1, value1, label2, value2)
	item1 := widget.NewAccordionItem("Address",
		grid)
	ac := widget.NewAccordion(item1)
	form.Append("", ac)
	w.SetContent(form)
	w.ShowAndRun()
}
```

![image](https://github.com/fyne-io/fyne/assets/37810104/63e68ac3-3280-45f5-b73b-79661929ff22)

**Comment by blaize** [comment]
Did you try to add the accordion when building the form widget ?
```
Items: []*widget.FormItem{
{Text: "Name", Widget: entry}, {Text: "Description", Widget: textArea}, {Text: "Address", Widget: ac}},
```

Another solution is to not use form widget, and build your own form logic with FormLayout

**Comment by andydotxyz** [comment, marked as answer]
The content of a form uses the pattern label (left) and content (fill right) - this is it's entire design.
If you want to put multiple parts of a form in elements of an accordion then you would need to split the form content into multiple Form widgets inserted into, or around, the accordion.

**Reply by pkk-code** [reply]
Thank you. I got working keeping them in multiple form layout with its own buttons.
![image](https://github.com/fyne-io/fyne/assets/37810104/788773c0-8859-40e7-9338-0eae7c934419)
